src/test/java/GS/test/GSBufferTests.java

```diff
@@ -298,32 +285,26 @@ void GSBuffer_WriteChannel_range()
             buffertest = new GSBuffer( 2000, 64);
         } catch (Exception ex) {fail(ex);}
 
-        try {
-            buffertest.appendValue(1,65);
-        } catch(Exception ex) {System.out.println("active channel positive range pass");}
-
-        try {
-            buffertest.appendValue(1,-10);
-        } catch(Exception ex) {System.out.println("active channel negative range pass");}
+        assertThrows(Exception.class, () -> buffertest.appendValue(1,65));
+        System.out.println("active channel positive range pass");
 
+        assertThrows(Exception.class, ()->  buffertest.appendValue(1,-10));
+        System.out.println("active channel negative range pass");
     }
 
     /**
      * attempt to write not in increasing order
      */
     @Test
-    void GSBuffer_WriteChannel_order()
+    void GSBuffer_WriteChannel_order() // TODO: rewrite this test
     {
         try {
             buffertest = new GSBuffer( 2000, 64);
             buffertest.appendValue(1,1);
             buffertest.appendValue(1,3);
         } catch (Exception ex) {fail(ex);}
 
-        try {
-            buffertest.appendValue(1,2);
-        } catch(Exception ex) {System.out.println("active channel out of order pass");}
-
+        assertThrows(ActiveChanException.class, () -> buffertest.appendValue(1,2));
     }
```

Comment:
add the "system.out.println('active channel out of order pass')"

Reply:
added.